### ISSUE TITLE
Component Doesn't render 

### ISSUE THREAD

**Issue by leedium** [leedium/occ-react-component#45]
Reported issues in OSX where component(s) are not rendering, and seeing the following issue 

503 Failed: Charles Connect: Connection refused (Connection refused)
```Uncaught Error: Script error for: {HOST}/file/v4070319364406092322/widget/occReactComponent/js/bundle.min.js
http://requirejs.org/docs/errors.html#scripterror
    at C (require.js?bust=18.6.1.1:1)
    at HTMLScriptElement.onScriptError (require.js?bust=18.6.1.1:1)
```

**Comment by leedium** [comment]
Tested on two MPBs running 
2014MPB -  OSX 10.14.2, NodeJS: 10.14.2, Java(build 1.8.0_191-b12)
2017MBP - OSX 10.14.2, NodeJS: 10.14.2, Java(build 1.8.0_191-b12)

Works on the 2014 and not the 2017.

When setting
```
https:false
``` 
And updating the proxy settings for the websockets, it work.  Will merge in a fix shortly.

**Comment by leedium** [comment]
Also tested with NodeJS: 9.9.0 and this is also failing no time to investigate but I do have a fix for versions < 10

I have added logic to toggle https based on the versions.
Please update you Charles or proxy mappings using the new xml configurations located in `charles remote configs/`

**Comment by leedium** [comment]
Pull request fix into 2.1.1
